portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,75 @@
 #content {
```

Comment:
Overall, commits with UI/UX changes should have links to screenshots with how the website looks. Makes it better for code reviews and just easy to visualize what changed per commit. Could you add a screenshot link?

Reply:
I do not have a screenshot link for before the changes, but here is what it currently looks like: https://screenshot.googleplex.com/tKeOLVPqhTM

Reply:
One of the most important reasons to use a tool like git is that your code history is always accessible.

This means you are able to check out an earlier version, get a screenshot from that version, and add it to this commit. Let us know if you need any help with that.

Also, we should find a screenshot solution that works outside Google as this is an open source project.